packages/docs-page/server/loaders/remote-content.ts

```diff
@@ -79,22 +97,18 @@ const cachedFetchVersionMetadataList = moize(
 export function mapVersionList(
   list: VersionMetadataItem[]
 ): VersionSelectItem[] {
-  const versions = list
-    .sort((a, b) =>
-      semver.compare(semver.coerce(b.version)!, semver.coerce(a.version)!)
-    )
-    .map((e) => {
-      const { isLatest, version, display } = e
-
-      const displayValue = display || version
+  const versions = list.map((versionOption) => {
+    const { isLatest, version, display, releaseStage } = versionOption
+    const displayValue = display || version
```

Comment:
💭 Given that `displayValue` is derived from `versionOption`, maybe you can lift this into `determineLabel(option)`, remove the second arg

Reply:
Good catch! Was wondering what to do with displayValue, but didn't think of this somehow!